src/ingest_validation_tools/table-schemas/includes/fields/transposition_transposase_source.yaml

```diff
@@ -0,0 +1,14 @@
+# bulkatacseq-v0.yaml; bulkatacseq-v1.yaml
+- name: transposition_transposase_source
+  description: The source of the Tn5 transposase and transposon used for capturing
+    accessible chromatin.
+
+
+# scatacseq-v0.yaml; scatacseq-v1.yaml
+- name: transposition_transposase_source
+  description: The source of the Tn5 transposase and transposon used for capturing accessible chromatin.
+  constraints:
+    enum:
+      - 10X snATAC
```

Comment:
This enum list is correct for sc/sn ATACseq only. Chris will follow-up on bulk ATACseq input for this field (review already submitted metadata for bulkATAcseq to check).